### PR TITLE
Fix for potentially disastrous file deleting bug.

### PR DESCRIPTION
**Pull request by hhanesand**

### CODE REVIEW

**Comment by hhanesand** [comment]
Will merge and close, but creating this for reference.

If you see anything amiss, let me know, but I am very certain I got it right this time. Also took a peek at the iOS filesystem documentation and decided to exclude the pictures we save on disk from iCloud backups.

File: Pod/Classes/ABEReporterViewController.swift [hunk 1]
```diff
@@ -97,7 +97,7 @@ internal class ABEReporterViewController: UIViewController {
     }
     
     func dismissIssueReporter() {
-        FileManager.clearDocumentsDirectory()
+        FileManager.earseStoredPicturesFromDisk()
```
Comment (heumn) [inline Pod/Classes/ABEReporterViewController.swift:100]: typo? :)

Comment (hhanesand) [reply Pod/Classes/ABEReporterViewController.swift:100]: yes

**Comment by heumn** [comment]
JMV7IKoqzxlrW

**Pull request merged**